<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="SR_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Bic_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SR_pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Jenson Pro Lt Disp</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>高分辨视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Play_pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Jenson Pro Lt Disp</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>同时播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bic_pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Jenson Pro Lt Disp</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>低分辨视频</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
